src/measure_theory/measure/measure_space.lean

```diff
@@ -3424,6 +3424,15 @@ piecewise_ae_eq_of_ae_eq_set hst
 lemma indicator_meas_zero (hs : μ s = 0) : indicator s f =ᵐ[μ] 0 :=
 (indicator_empty' f) ▸ indicator_ae_eq_of_ae_eq_set (ae_eq_empty.2 hs)
 
+lemma indicator_ae_eq_zero_of_ae_eq_zero (hf : f =ᵐ[μ] 0) : s.indicator f =ᵐ[μ] 0 :=
```

Comment:
This should be a consequence of a similar lemma for any two functions, taking `f =ᵐ[μ] g` as hypothesis. Do we have it?

Reply:
I couldn't find it so I've added it at the same place